exercises/rna-transcription/src/example.c

```diff
@@ -0,0 +1,27 @@
+#include <stdlib.h>
+#include <string.h>
+#include "rna_transcription.h"
+
+char *to_rna(const char *dna) {
+  int len = strlen(dna);
+  char *rna = (char *)malloc(sizeof(char) * len);
```

Comment:
Can you remove this unnecessary cast? See http://stackoverflow.com/questions/605845/do-i-cast-the-result-of-malloc.